docs/manual/system/settings.de.md

```diff
@@ -525,6 +525,114 @@ Beschreibung.
 | `versionPeriod` | Zeitspanne in Sekunden wie lange ältere Versionen von geänderten Einträgen behalten werden sollen. Standard: `7776000`. |
 
 
+## Umgebungsvariablen
+
+{{< Version "4.9" >}}
+
+Die Datei `.env` muss sich im Hauptverzeichnis der Contao-Installation befinden. Falls eine zusätzliche `.env.local` Datei existiert, wird diese automatisch verwendet.
```

Comment:
The `.env` file isn't explained yet at this point. This section should probably explain first what environment variables are and how you can define them.

Reply:
I found the explanatory introduction from the dev doc too "technical" for the manual and therefore didn't use it at first.

Reply:
Well but your first sentence then requires the reader to know what a `.env` file is - so surely this isn't better? ;)

Reply:
Yepp - An attempt with the further commits